uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/_variables.scss

```diff
@@ -0,0 +1,7 @@
+@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables";
+@import "SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/variables";
+
+$chr-table-condensed-cell-padding: 8px 5px;
```

Comment:
This is a standard `_variables.scss` partial that any theme (local or global) has, also these variables are used across the board, in mixins, overriding styles, components. It's better to keep them here, i think